src/main/startSandbox.js

```diff
@@ -0,0 +1,28 @@
+import express from 'express';
+import unpackByOutpoint from './unpackByOutpoint';
+
+// Polyfills and `lbry-redux`
+global.fetch = require('node-fetch');
+
+global.window = global;
+// eslint-disable-next-line import/no-commonjs,global-require
+const { Lbry } = require('lbry-redux');
+
+delete global.window;
+
+export default async function startSandbox() {
+  const sandbox = express();
+  const port = 5278;
+
+  sandbox.get('/set/:outpoint', async (req, res) => {
+    const { outpoint } = req.params;
+    const resolvedPath = await unpackByOutpoint(Lbry, outpoint);
+
+    sandbox.use(`/sandbox/${outpoint}/`, express.static(resolvedPath));
+
+    res.send(`/sandbox/${outpoint}/`);
```

Comment:
Is this needed?

Reply:
Yes, see `src/renderer/component/fileViewer/internal/player.jsx`, it uses the response to know where to go when the sandbox is ready.